### PR TITLE
Create document fetching

### PR DESCRIPTION
**Pull request by TheBroMoe**
* Replaced static about page with dynamic escaped html from ElasticSearch
* Created front end get request to receive decoded html page

### CODE REVIEW

**Comment by TheBroMoe** [comment]
> Hi @TheBroMoe thanks for your contribution. I think you found an unrelated bug (second last comment). Perhaps you wouldn't mind cleaning it up in this pr?
> 
> I wonder if this couldn't be extended with some type of interface in the admin section for generating the document. Not sure if it's a priority, but I do think that this `PR` will make the `/about` section cumbersome to edit. The `dynamicDocument` editor could tie in with the Rich Text Editor (future) implementation. Perhaps you've been thinking about this already.

I'll be sure to look into everything you mentioned. Thank you for the Feedback! I'll resend a PR when I've made all the changes.

File: frontend/src/app/user/about/about.service.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    constructor(private http: Http) {
+        console.log('Creating aboutservice')
+        this.documents =  this.http.get(`${environment.api}/api/document/about`).toPromise();
+    }
+
+
+    async getDocument(): Promise<string>  {
+        const httpRes = await this.documents;
+        return decodeURIComponent(httpRes._body);
```
[the remark this answers is not in the document]
Comment (TheBroMoe) [reply frontend/src/app/user/about/about.service.ts]: We started by testing to see if we could send a post with encoded HTML and wanted to return a decoded post, and passing regular html returns the same body. If we never plan on sending encoded html we can remove this call

[the remark this answers is not in the document]
Comment (slmyers) [reply frontend/src/app/user/about/about.service.ts]: Oh, I see. That's fine. Probably more secure.

**Comment by TheBroMoe** [comment]
Everything mentioned should now be resolved. Be sure to look over the request again and let me know if there are any changes I should make. Thanks!

File: frontend/src/app/user/about/about.service.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    constructor(private http: Http) {
+        this.documents =  this.http.get(`${environment.api}/api/document/about`).toPromise();
+    }
+
+
+    async getDocument(): Promise<string>  {
+        const httpRes = await this.http.get(`${environment.api}/api/document/about`).toPromise();
+        
+        // const httpRes = await this.documents;
```
Comment (j-rewerts) [inline frontend/src/app/user/about/about.service.ts]: Remove this comment.

File: backend/src/modules/document/document.service.ts [hunk 3]
```diff
@@ -5,8 +5,9 @@ import { Observable } from "rxjs/Observable";
 import "rxjs/add/observable/fromPromise"
 import "rxjs/add/operator/map"
 import "rxjs/add/operator/do"
+import * as sanitizeHtml from "sanitize-html";
 const uuidv4 = require("uuid/v4");
-
+console.log(sanitizeHtml)
```
Comment (j-rewerts) [inline backend/src/modules/document/document.service.ts]: Remove this line.

File: backend/src/modules/api/api.controller.ts [hunk 4]
```diff
@@ -30,6 +31,11 @@ export class ApiController {
         return this.programService.getByGuid(params.guid);
     }
 
+    @Get('document/:guid')
+    getDocumentByGuid(@Param() params): Promise<string>{
+        return this.documentService.getByTitle(params.guid);
```
Comment (CodyGramlich) [inline backend/src/modules/api/api.controller.ts]: Hi @TheBroMoe, it looks like you forgot to change params.guid to params.title and getDocumentByGuid to getDocumentByTitle.

Comment (TheBroMoe) [reply backend/src/modules/api/api.controller.ts]: Hi @CodyGramlich, Thanks for letting me know, I'll change it right away!

**Comment by j-rewerts** [comment]
Closing in lieu of #108.

**Pull request closed without merging**